cfgov/unprocessed/apps/research-reports/css/main.less

```diff
@@ -29,14 +36,6 @@
     color: #101820;
     border-color: #20aa3f;
   }
-  .o-secondary-navigation_list__children {
-    display: none;
-  }
-  .parent-header {
-    .o-secondary-navigation_list__children  {
-      display: block;
-    }
-  }
```

Comment:
This code controls the logic of only displaying children of the current section in the ToC, to help prevent it from being extraordinarily wrong. It's possible we'll want to change this behavior for just the mobile ToC, but it needs to stay in the desktop view.